core/components/tree/basenode.js

```diff
@@ -731,14 +722,14 @@ Blockly.tree.BaseNode.prototype.onKeyDown = function(e) {
   var handled = true;
   switch (e.keyCode) {
     case Blockly.utils.KeyCodes.RIGHT:
-      if (e.altKey) {
+      if (e.altKey) {  // Why?
```

Comment:
No clue, doing a blame on the closure code didn't yield any insight, that bit of code was there from 11 years ago for when it was moved over to git.

I would say remove.

Reply:
Removed.